nnef/src/ops/nnef/deser.rs

```diff
@@ -449,10 +504,44 @@ pub fn matmul(
     builder: &mut ModelBuilder,
     invocation: &ResolvedInvocation,
 ) -> TractResult<TVec<OutletId>> {
-    let a = invocation.named_arg_as(builder, "A")?;
-    let b = invocation.named_arg_as(builder, "B")?;
+    let a: OutletId = invocation.named_arg_as(builder, "A")?;
+    let b: OutletId = invocation.named_arg_as(builder, "B")?;
     let a_trans = invocation.named_arg_as(builder, "transposeA")?;
     let b_trans = invocation.named_arg_as(builder, "transposeB")?;
+    if let Some(Some(dt)) = &invocation.dt.get(0) {
+        if let Some(qparams) = dt.qparams() {
+            let (a0, a_scale) =
+                builder.model.node(a.node).outputs[a.slot].fact.datum_type.zp_scale();
+            let (b0, b_scale) =
+                builder.model.node(b.node).outputs[b.slot].fact.datum_type.zp_scale();
+
+            let (c0, c_scale) = qparams.zp_scale();
+            //FIXME: bias is not specified in the nnef format, but whether the bias is a bias or an add later changes the quantized behaviour
```

Comment:
Again, just to highlight this

Reply:
we need a tract_core_matmul_with_bias (or an extra parameter to matmul ?)